README.md

```diff
@@ -150,6 +150,16 @@ A command-line flag exists `--dnsEndpoints` which can optionally include a comma
 - Check name: `dnsStatus`
 
 
+#### Deployment and Service
+
+Deploys a `deployment` to the `kuberhealthy` namespace with `2` replicas, waits for pods and service to be in the 'Ready' state and serving a hostname, then makes a request to the endpoint for a `200 OK`, then terminates them and ensures deployment and service terminations were successful.  Containers are deployed with their resource requirements set to 15 millicores and 20Mi units of memory.  The check uses the latest Nginx web server container (`nginx:latest`) for the deployment.  A success means that a deployment and service can be brought up and that the corresponding hostname endpoint can return a `200 OK`.  If `CHECK_DEPLOYMENT_ROLLING_UPDATE` is set to `true`, the check will initially deploy Nginx's alpine web server container and roll to the latest (`nginx:alpine` -> `nginx:latest`) and look for a second `200 OK`.  A failure means that an error occurs anywhere in the deployment creation, service creation, HTTP request, and tear down resulting in an error report to Kuberhealthy.
```

Comment:
--> The check deploys a `deployment` to the `kuberhealthy` namespace with `2` replicas and waits for both the pods and service to be in `Ready` state and serve a hostname. Once ready, the check makes a request to the endpoint for a `200 OK` response and then proceeds to terminate them and ensures that the deployment and service terminations were successful. 

--> A successful check run implies that a deployment and service can be brought up and the corresponding hostname endpoint returns a `200 OK` response. 

--> and look for a second `200 OK` response. 

--> A failure implies that an error occurred anywhere in the deployment creation, service creation, HTTP request, or tear down process, resulting in an error report to the Kuberhealthy status page.